llvm/test/CodeGen/RISCV/GlobalISel/jumptable.ll

```diff
@@ -0,0 +1,263 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; RUN: llc -mtriple=riscv32 -code-model=small -verify-machineinstrs < %s \
+; RUN:   -global-isel | FileCheck %s -check-prefixes=RV32I-SMALL
+; RUN: llc -mtriple=riscv32 -code-model=medium -verify-machineinstrs < %s \
+; RUN:   -global-isel | FileCheck %s -check-prefixes=RV32I-MEDIUM
+; RUN: llc -mtriple=riscv32 -relocation-model=pic -verify-machineinstrs < %s \
+; RUN:   -global-isel | FileCheck %s -check-prefixes=RV32I-PIC
+; RUN: llc -mtriple=riscv64 -code-model=small -verify-machineinstrs < %s \
+; RUN:   -global-isel | FileCheck %s -check-prefixes=RV64I-SMALL
+; RUN: llc -mtriple=riscv64 -code-model=medium -verify-machineinstrs < %s \
+; RUN:   -global-isel | FileCheck %s -check-prefixes=RV64I-MEDIUM
+; RUN: llc -mtriple=riscv64 -relocation-model=pic -verify-machineinstrs < %s \
+; RUN:   -global-isel | FileCheck %s -check-prefixes=RV64I-PIC
+
+define void @above_threshold(i32 signext %in, ptr %out) nounwind {
```

Comment:
This test is a copy of a llvm/test/CodeGen/RISCV/jumptable.ll  I diffed the output of both and didn't spot any differences other than instruction order.